newsteller.sh

```diff
@@ -1,9 +1,39 @@
-wget -o jump.txt  -O index.html www.mangareader.net
-grep 'chaptersrec' index.html > jump.txt
-while read p;do
-  starti="$(echo $p | grep -aob '>' | grep -oE '[0-9]+' | sed "1q;d")"
-  endi="$(echo $p | grep -aob '<' | grep -oE '[0-9]+' | sed "2q;d")"
-  length=$((endi-starti))
-  newchap=${p:$((starti+1)):$((length-1))}
-  echo $newchap
-done < jump.txt
+getLatestInfo () {
+  manga=$1
+  #Gets the manga's html page
+  wget -O temp.html -q www.mangareader.net/$manga
+  #Finds the string between the patterns '<a href="/manga/" and '</a>' at first occurence
+  latest=$(grep -m1 -oP "(?<=a href=\"/${manga}/).*(?=</a>)" temp.html)
+  #remove all characters before '>' in string latest including '>'
+  chapl=$(echo $latest | sed -e 's/.*>//g')
+  #gets the line number of last occurence of string chapl
+  lnum=$(cat -n temp.html | grep "$chapl" | tail -1 |  cut -f 1)
+  #increment lnum by 1
+  ((lnum++))
+  #get the content in the line lnum
+  date=$(sed "${lnum}q;d" temp.html)
+  #remove the first four and last 5 characters from string date
+  date=${date:4:-5}
+  #delete temp.html
+  rm temp.html
+  printf "%s %s\n" "$chapl" "$date"
```

Comment:
For a better table view, you might want to write it as "printf %25s %25s\n" Or something of that.
The best way would be, acc. to me., The chapter name to the very left, chapter number somewhere in the middle and the date to the extreme right. Check up printf functionality for it :smile: